modules/test/playwright/pages/portal-search-web/SearchPage.ts

```diff
@@ -0,0 +1,165 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {FrameLocator, Locator, Page, expect} from '@playwright/test';
+
+import {clickAndExpectToBeVisible} from '../../utils/clickAndExpectToBeVisible';
+
+export class SearchPage {
+	readonly page: Page;
+	readonly modalIFrame: FrameLocator;
+	readonly searchBarInput: Locator;
+	readonly searchBarInputInNavBar: Locator;
+	readonly searchOptionsConfigurationLink: Locator;
+	readonly searchResults: Locator;
+	readonly searchResultsItems: Locator;
+	readonly searchResultsPaginationBar: Locator;
+	readonly searchResultsPaginationDescription: Locator;
+	readonly searchResultsPaginationItemsPerPageToggle: Locator;
+	readonly searchResultsPaginationItemsPerPageDropdown: Locator;
+	readonly searchResultsTotalLabel: Locator;
+
+	constructor(page: Page) {
+		this.page = page;
+
+		this.modalIFrame = page.frameLocator('iframe[id="modalIframe"]');
+		this.searchBarInput = page
+			.locator('#main-content')
+			.getByPlaceholder('Search...');
+		this.searchBarInputInNavBar = page
+			.locator('.navbar')
+			.getByPlaceholder('Search...');
```

Comment:
To clearly differentiate between these 2, I'd name the first one `this.searchBarInputInMainContent`.